service/worker/scheduler/activities.go

```diff
@@ -65,6 +68,8 @@ type (
 	}
 )
 
+const maxResultSize = 2 * 1024 * 1024
```

Comment:
get this from dynamic config: https://github.com/temporalio/temporal/blob/main/common/dynamicconfig/constants.go#L123-L124